prioritizer/prioritizer.py

```diff
@@ -27,58 +29,17 @@
 # File paths
 OUTPUT_FILE = "new_output.json"
 
-def send_to_llm(prompt):
-    response = client.chat.completions.create(
-        model="gpt-4o",
-        messages=[
-            {
-                "role": "system",
-                "content": """
-            You are a climate action expert, tasked to prioritize climate actions for cities.
-             
-            These are the rules to use for prioritizing a climate action for a city:
-             
-            - Lower cost actions are better than higher cost actions.
-            - High emissions reductions are better than low emissions reductions.
-            - High risk reduction is better than low risk reduction.
-            - Actions that match the environment are better than those that don't.
-            - Actions that match the population are better than those that don't.
-            - Actions that take less time are better than those that take more time.
-            """,
-            },
-            {"role": "user", "content": prompt},
-        ],
-    )
-    return response.choices[0].message.content
 
 def read_city_inventory():
```

Comment:
read_city_inventory and read_actions are also two functions that I would put into a separate /utils folder.
They are just data loaders and are not directly concerned with the prioritization. having them in separate files makes the actual scoring file much leaner and shorter